README.md

```diff
@@ -487,6 +489,10 @@ care of lower/uppercases**) : `yaml: a.b --> envvar: A_B` :
   `false`)
 - **ELASTICSEARCH_CHECKCERT** : check if ssl certificate of the output is valid (default:
   `true`)
+- **ELASTICSEARCH_USERNAME** : use this username to authenticate to Elasticsearch if the
+  username is not empty (default: "")
+- **ELASTICSEARCH_PASSWORD** : use this password to authenticate to Elasticsearch if the
+  password is not empty (default: "")
```

Comment:
can you sync descriptions between files please

Reply:
Sure - which description would you prefer?

Reply:
`use this (username|passworrd) to authenticate to Elasticsearch if not empty (default: "")`

seems good to me, wdyt?

Reply:
Ok, I think I got to this. Let me know if you're not happy with the wording. I can change it. :)